src/main/fc/rc.c

```diff
@@ -53,13 +53,20 @@
 
 typedef float (applyRatesFn)(const int axis, float rcCommandf, const float rcCommandfAbs);
 
+#ifdef USE_INTERPOLATED_SP
+// Setpoint in degrees/sec before RC-Smoothing is applied
+static float rawSetpoint[XYZ_AXIS_COUNT];
```

Comment:
But the computation is pretty involved. It seems one wouldn't want to calculate it twice when a new rc datapoint arrives.

Reply:
OK .. 
theoretically, local difference is only needed, so it may be possible to implement function that would return derivative at given point. But that's overkill in current BF state ...